src/sentry/static/sentry/app/views/organizationDiscover/earlyAdopterMessage.jsx

```diff
@@ -12,7 +12,9 @@ export default class EarlyAdopterMessage extends React.Component {
     return (
       <StyledAlert type="info" icon="icon-labs">
         <StyledAlertText>
-          {t('Sentry Discover is alpha software. Thanks for being an early adopter.')}&nbsp;
+          {t(
+            'Sentry Discover is alpha software. Thanks for being an early adopter.'
```

Comment:
is this the prettier versions again?

Reply:
I have the same prettier as in `package.json`: 1.7.4, so I think it just wasn't formatted properly initially.